rules/prettier.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: ['prettier'],
-  extends: ['prettier', 'prettier/@typescript-eslint'],
```

Comment:
Are we not supposed to use 'prettier/@typescript-eslint'?

Reply:
which check in `rules/typescript.js`? It does use 'prettier/@typescript-eslint', but since version 8 of the `eslint-config-prettier` you don't need to manually extend it: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

Reply:
Thanks for the clarification :)